api/client/utils.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 var (
-	errConnectionRefused = errors.New("Cannot connect to the Docker daemon. Is 'docker -d' running on this host?")
+	errConnectionRefused = errors.New("Cannot connect to the Docker daemon. Is the docker daemon running on this host?")
```

Comment:
Yep, I couldn't think of a good name to change it to... I don't think errConnectionFailed is a good name either unfortunately.